src/main/java/com/stripe/model/oauth/TokenResponse.java

```diff
@@ -0,0 +1,98 @@
+package com.stripe.model.oauth;
+
+import com.stripe.model.StripeObject;
+
+public class TokenResponse extends StripeObject {
+	Boolean livemode;
+	String scope;
+	String stripeUserId;
+
+	@Deprecated
+	String accessToken;
+	@Deprecated
+	String refreshToken;
+	@Deprecated
+	String stripePublishableKey;
+	@Deprecated
+	String tokenType;
+
+	public Boolean getLivemode() {
+		return livemode;
+	}
+
+	public void setLivemode(Boolean livemode) {
+		this.livemode = livemode;
+	}
+
+	public String getScope() {
+		return scope;
+	}
+
+	public void setScope(String scope) {
+		this.scope = scope;
+	}
+
+	public String getStripeUserId() {
+		return stripeUserId;
+	}
+
+	public void setStripeUserId(String stripeUserId) {
+		this.stripeUserId = stripeUserId;
+	}
+
+	/**
+	 * @deprecated Use `stripe_user_id` with the `Stripe-Account` header (https://stripe.com/docs/connect/authentication#stripe-account-header)
+	 */
```

Comment:
Is the idea to bring in deprecated fields for older API versions? I wonder if we should wait until someone complains ... :)

Reply:
Yeah, I was being conservative here, but since this is a new feature as far as the Java library is concerned, we might as well not declare the deprecated fields at all. I'll update the PR.